### PR TITLE
ENG-3683 amend svc list command

### PR DESCRIPTION
**Pull request by jeffgo10**

### CODE REVIEW

File: src/commands/svc/list.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    const svcService: SvcService = new SvcService(process.cwd())
+    const services: string[] = flags.available
+      ? svcService.getAvailableServices()
+      : svcService.getActiveServices()
+    const serviceData = services.map(service => ({ service }))
+
+    const columns = {
+      service: {
+        header: flags.available ? 'Available Services' : 'Active Services'
```
Comment (cecchisandrone) [inline src/commands/svc/list.ts]: Active Services -> Enabled Services

File: src/commands/svc/list.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      }
```
Comment (cecchisandrone) [inline src/commands/svc/list.ts:33]: After listing the available services I would like to add a message like this: "Hint: you can add an auxiliary service just dropping a Docker Compose file in the svc folder (e.g. svc/my-service.yml)"

File: src/services/svc-service.ts [hunk 3]
```diff
@@ -29,10 +29,20 @@ export class SvcService {
       .map(filename => filename.slice(0, -4))
```
Comment (cecchisandrone) [inline src/services/svc-service.ts:29]: Always better to filter specifically for files ending with .yml

Comment (jeffgo10) [reply src/services/svc-service.ts:29]: @cecchisandrone there's a filter function above that exactly does what you mentioned. the `map` only extracts the filenames, hence removing extension types.

File: src/commands/svc/list.ts (hunk 1 above)
Comment (jeffgo10) [reply src/commands/svc/list.ts]: done

File: src/commands/svc/list.ts (hunk 2 above)
Comment (jeffgo10) [reply src/commands/svc/list.ts:33]: log added at the end of table listing

**Review by cecchisandrone — APPROVED** [review]

**Pull request merged**